src/BestOfTechnozaure/Talk/LogoDate.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+	AbsoluteFill,
+	Img,
+	spring,
+	staticFile,
+	useCurrentFrame,
+	useVideoConfig,
+} from 'remotion';
+import {Lottie} from '@remotion/lottie';
+import React from 'react';
+import {useLottie} from '../../Components/hooks/useLotttie';
+
+export const LogoDate: React.FC<{date: string}> = ({date}) => {
+	const frame = useCurrentFrame();
+	const {fps} = useVideoConfig();
+
+	const illustrationDate = useLottie('lf20_ak90tqhe');
+
+	const durationInFrames = 40;
+	const drop = spring({frame, from: -20, to: 0, fps, durationInFrames});
+	const opacity = spring({frame, from: 0, to: 1, fps, durationInFrames});
+
+	if (!illustrationDate) {
+		return null;
+	}
+
+	return (
+		<AbsoluteFill>
+			<hr
+				style={{
+					position: 'absolute',
+					width: '5px',
+					height: '80px',
+					border: 'none',
+					background: 'white',
+					left: '50%',
+					bottom: 70,
+					transform: 'translateX(-50%)',
+					opacity,
+				}}
+			/>
```

Comment:
I think that the `hr` should be independent and not in this component 😁

Reply:
You want me to create a new component "vertical divider" ?